apps/dcos_l4lb/src/dcos_l4lb_mesos_poller.erl

```diff
@@ -88,24 +89,40 @@ handle_poll() ->
 handle_poll(false) ->
     ok;
 handle_poll(true) ->
+    Begin = erlang:monotonic_time(),
     try dcos_net_mesos_listener:poll() of
         {error, Error} ->
+            prometheus_counter:inc(
+                l4lb, poll_failures_total,
+                [], 1),
             lager:warning("Unable to poll mesos agent: ~p", [Error]);
         {ok, Tasks} ->
             handle_poll_state(Tasks)
     catch error:bad_agent_id ->
         lager:warning("Mesos agent is not ready")
+    after
+        prometheus_summary:observe(
+            l4lb, poll_request_duration_seconds,
+            [], erlang:monotonic_time() - Begin)
     end.
 
 -spec(handle_poll_state(#{task_id() => task()}) -> ok).
 handle_poll_state(Tasks) ->
-    Tasks0 = maps:filter(fun is_healthy/2, Tasks),
-
-    PortMappings = collect_port_mappings(Tasks0),
+    Begin = erlang:monotonic_time(),
+    HealthyTasks = maps:filter(fun is_healthy/2, Tasks),
+    prometheus_gauge:set(l4lb, local_tasks, [], maps:size(Tasks)),
+    prometheus_gauge:set(
+        l4lb, local_healthy_tasks,
+        [], maps:size(HealthyTasks)),
+
+    PortMappings = collect_port_mappings(HealthyTasks),
     dcos_l4lb_mgr:local_port_mappings(PortMappings),
 
-    VIPs = collect_vips(Tasks0),
-    ok = push_vips(VIPs).
+    VIPs = collect_vips(HealthyTasks),
+    ok = push_vips(VIPs),
+    prometheus_summary:observe(
+        l4lb, poll_process_duration_seconds,
```

Comment:
nit: `process -> processing/handling`?

Reply:
I think the poposed change here is to rename this group of metrics to:
`
l4lb_call_received_bytes_total
l4lb_call_duration_seconds
l4lb_call_failures
`

would that be enough for you? They were also all moved to `dcos_net_mesos`

Reply:
ditto; feel free to reopen.